spring-cloud-skipper-docs/src/main/asciidoc/configuration-database.adoc

```diff
@@ -0,0 +1,65 @@
+
+[[configuration-database-overview]]
+A relational database is used to store stream and task definitions as well as the state of executed tasks.
```

Comment:
Do we want to mention tasks in skipper?   It has no reference to them.   If we run in Stream-Only mode those tables are not used.

Reply:
Yep, good catch.... it was like that [before](https://github.com/spring-cloud/spring-cloud-skipper/pull/1061/files#diff-a7435791a7ff08ddcbdbb5c68fb777d69c91e8600d8340f02900eb5193e92a7bL223) but it should only talk about streams here for sure. I will adjust.